doc/containers.md

```diff
@@ -360,11 +365,17 @@ minor       | int       | device on host    |
 uid         | int       | 0                 |                                   | no        | UID of the device owner in the container
 gid         | int       | 0                 |                                   | no        | GID of the device owner in the container
 mode        | int       | 0660              |                                   | no        | Mode of the device in the container
+required    | boolean   | true              | unix\_device\_hotplug             | no        | Whether or not this device is required to start the container.
 
 ### Type: unix-block
 Unix block device entries simply make the requested block device
 appear in the container's `/dev` and allow read/write operations to it.
 
+If they exist at the time of the LXD daemon start the "required" property is
+available for devices in the following directories: `/dev`, `/dev/dri`,
+`/dev/infiniband`, `/dev/input`, `/dev/mapper`, `/dev/net`, `/dev/pts`,
+`/dev/raw`, `/dev/thinkpad`, `/dev/scanners`, `/dev/usb`, `/dev/video`.
```

Comment:
Why do we still have this comment (and the same for unix-char)?
The new approach doesn't restrict us to specific directories anymore right?

Reply:
removed

Reply:
damn, only removed in from `unix-char` :)

Reply:
addressed